app/src/main/java/com/example/recipeapp/Fragments/SuggestFragment.java

```diff
@@ -0,0 +1,108 @@
+package com.example.recipeapp.Fragments;
+
+import android.content.Intent;
+import android.os.Bundle;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.fragment.app.Fragment;
+import androidx.lifecycle.Observer;
+import androidx.lifecycle.ViewModelProviders;
+
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Toast;
+
+import com.example.recipeapp.Activities.DetailsActivity;
+import com.example.recipeapp.Adapters.AutoCompleteAdapter;
+import com.example.recipeapp.Adapters.IngredientFilterAdapter;
+import com.example.recipeapp.Adapters.IngredientsAdapter;
+import com.example.recipeapp.Adapters.StepsAdapter;
+import com.example.recipeapp.BuildConfig;
+import com.example.recipeapp.Models.Ingredient;
+import com.example.recipeapp.Models.Recipe;
+import com.example.recipeapp.R;
+import com.example.recipeapp.Repositories.RecipesRepository;
+import com.example.recipeapp.Retrofit.RecipeApi;
+import com.example.recipeapp.Retrofit.RetrofitClientInstance;
+import com.example.recipeapp.databinding.FragmentSuggestBinding;
+import com.example.recipeapp.viewmodels.LoginViewModel;
+import com.example.recipeapp.viewmodels.SuggestViewModel;
+import com.google.android.flexbox.FlexDirection;
+import com.google.android.flexbox.FlexWrap;
+import com.google.android.flexbox.FlexboxLayoutManager;
+
+import org.parceler.Parcels;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import retrofit2.Call;
+import retrofit2.Callback;
+import retrofit2.Response;
+
+public class SuggestFragment extends Fragment {
+
+    private static final String TAG = "Suggest activity";
+    FragmentSuggestBinding binding;
+    IngredientFilterAdapter iHaveAdapter;
+    SuggestViewModel viewModel;
```

Comment:
I have not seen these used outside `onViewCreated `, we can avoid adding these params if not needed.